scripts/jenkins/groovy/defineTestStages.groovy

```diff
@@ -45,6 +45,10 @@ def call(final pipelineContext) {
       stageName: 'Py3.10 Smoke', target: 'test-py-smoke', pythonVersion: '3.10', timeoutValue: 8,
       component: pipelineContext.getBuildConfig().COMPONENT_PY
     ],
+    [
+      stageName: 'Py3.11 Smoke', target: 'test-py-smoke', pythonVersion: '3.11', timeoutValue: 8,
```

Comment:
I think smoke stages was only for Lowest and Highest version.

Reply:
Yes, good catch! I forgot about to remove python 3.10